.NET/Microsoft.Recognizers.Text.DateTime/English/Extractors/EnglishTimeExtractorConfiguration.cs

```diff
@@ -140,10 +145,16 @@ public class EnglishTimeExtractorConfiguration : ITimeExtractorConfiguration
 
         IEnumerable<Regex> ITimeExtractorConfiguration.TimeRegexList => TimeRegexList;
 
+        public IExtractor TimeExtractor => new BaseTimeExtractor(new English.EnglishTimeExtractorConfiguration());
+
+        public IExtractor NumExtractor => NumberExtractor.GetInstance();
```

Comment:
These should be in the construction method. Please follow the initialization of the durationExtractor in this file.